en/docs/guides/organization-login/try-organization-login.md

```diff
@@ -155,9 +155,9 @@ By default the **Username & Password** authentication step is added to the Sign-
 
 To set up the sample application:
 
-1. Download the [sample application](https://github.com/Achintha444/guardio-insurance-sample-application/archive/main.zip), and extract the content of the zip file.
+1. Download the [sample application](https://github.com/wso2/samples-is/archive/refs/heads/master.zip), and extract the content of the zip file.
```

Comment:
We don't need to download the whole zip for all the samples, instead can't we just download only the B2B Sample?
Otherwise, it might be confusing for the user and it's not necessary either.

Reply:
> Download the sample for playground2.war

> doc: https://is.docs.wso2.com/en/latest/guides/access-delegation/implicit-playground/#download-the-sample
> download link: https://github.com/wso2/samples-is/releases/download/v4.5.0/playground2.war

Reply:
Try this type of approach.